clustering/kmeans_sample.ipynb

```diff
@@ -0,0 +1,518 @@
+{
```

Comment:
<ul><li>maybe change the sample points to a 2D array rather than two 1D array? Then you don't have to apply <code>np.stack()</code> in the next part.</li><li>nit: use <code>np.random.random()</code> instead of <code>np.random.randint()</code>. I cannot find any reason to use <code>int</code> instead of <code>float</code> here</li></ul><p> </p><p><code>n_dim = 2</code></p><p><code>scale = (200, 100)</code></p><p><code>points = np.random.random((M, n_dim)) * scale</code></p><p><code>centroids = np.random.random((K, n_dim)) * scale</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Women-in-Data-Science-Taipei/2020-WiDS-Taipei-MLCC-Workshop/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>